### ISSUE TITLE
Signin view throws exception under Django 2.1 due to new mandatory argument on is_safe_url

### ISSUE THREAD

**Issue by atkinsj** [fangli/django-saml2-auth#53]
[is_safe_url is used in the signin view](https://github.com/fangli/django-saml2-auth/blob/master/django_saml2_auth/views.py#L202) but Django 2.1 recently [just made](https://docs.djangoproject.com/en/2.1/releases/2.1/) the allowed_hosts a mandatory parameter for this function.

You'll likely need to add a django.VERSION check and then push through settings.ALLOWED_HOSTS in kwargs if Django is >=2.1.

**Comment by aiic03** [comment]
Same issue here with Django 2.1

**Comment by fangli** [comment]
it was fixed in commit 90a2409, after I reviewed their source code of 1.8/1.9/2.x. 
please pull the master branch and reinstall it by
`python setup.py install`?
I'm unable to setup a django 2.x env at this moment, could you let me know your test result? If it works, I'll release it to pypi.

**Comment by mwoodbri** [comment]
90a2409 resolved the issue for me. Thanks! Looking forward to new release.

**Comment by joshowen** [comment]
@fangli do you have a planned next release?  We'd like to pickup this change as well.  Thank you!

**Comment by psavoie** [comment]
Hey @fangli will this fix be released soon?

**Comment by maxg203** [comment]
@fangli We're also waiting on this to be released. Would you be able to do that?

**Comment by maxg203** [comment]
If you're using Docker, here is the quick workaround that we're using until the next release:
```
ENV BAD_FILE /usr/lib/python3.6/site-packages/django_saml2_auth/views.py
RUN sed '202s/next_url/next_url, None/' $BAD_FILE > GOOD_FILE
RUN cat GOOD_FILE > $BAD_FILE
```

**Comment by fangli** [comment]
Will schedule a release today, thanks.

Max Goodridge <notifications@github.com>于2019年1月22日 周二下午9:56写道：

**Comment by fangli** [comment]
2.2.1 released with django 2.1 support.
pypi 2.2.1 released too.
